cmd/rule-evaluator/main.go

```diff
@@ -405,7 +411,7 @@ func (opts *evaluatorOptions) setupFlags(a *kingpin.Application) {
 		URLVar(&opts.TargetURL)
 
 	a.Flag("query.generator-url", "The base URL used for the generator URL in the alert notification payload. Should point to an instance of a query frontend that accesses the same data as --query.target-url.").
-		PlaceHolder("<URL>").
+		Default(googleCloudBaseURL.String()).
```

Comment:
@TheSpiritXIII could you update the docs for the [OperatorConfig](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/55d40dccb88e518b8e4e705dedb94191edce12fb/pkg/operator/apis/monitoring/v1/operator_types.go#L152) highlighting this is now the default?